examples/demo/demo.odin

```diff
@@ -2145,12 +2145,6 @@ or_return_operator :: proc() {
 			return -345 * z, zerr
 		}
 
-		// If the other return values need to be set depending on what the end value is,
-		// the 'defer if' idiom is can be used
-		defer if err != nil {
-			n = -1
```

Comment:
I would have just changed this specific line to something like `fmt.println("an error has occured", err)` or whatever, and not completely remove the entire thing.